src/app/modules/form/form.utils.ts

```diff
@@ -121,3 +123,8 @@ export const getLogicById = (
 
   return form_logics.find((logic) => logicId === String(logic._id)) ?? null
 }
+
+// Typeguard to check if a form has a message service id
+export const isOnboardedForm = (form: IForm): form is IOnboardedForm<IForm> => {
```

Comment:
nit: make arg more generic to allow better typing for forms that extend from `IForm`:

```ts
export const isOnboardedForm = <T extends IForm = IForm>(form: T): form is IOnboardedForm<T> => {
   ...
}
```

Reply:
yep, good point; changed!